rdt-app/src/main/java/io/ona/rdt_app/application/RDTApplication.java

```diff
@@ -56,7 +64,17 @@ public void onCreate() {
 
         getRepository();
 
-        JobManager.create(this).addJobCreator(new RDTJobCreater());
+        JobManager.create(this).addJobCreator(new RDTJobCreator());
+
+        getContext().userService(); // todo: can be removed when login screen is added
+
+        allSharedPreferences = getContext().allSharedPreferences();
+        initializeSharedPreferences(); // todo: can be removed when login screen is added
+
+        scheduleJobsPeriodically();
+
+        // todo: remove this
```

Comment:
Can this be removed instead of being commented

Reply:
Done.